include/miopen/miopen.h

```diff
@@ -112,11 +112,13 @@ typedef enum
     miopenStatusVersionMismatch = 10, /*!< Version mismatch of the supplied binary data argment. */
 } miopenStatus_t;
 
+#ifdef MIOPEN_BETA_API_FP8
```

Comment:
@cderb Right now we already have `MIOPEN_BETA_API` which controls LayerNorm. Adding `MIOPEN_BETA_API_FP8` for controlling F8 convolutions seems inconsistent.

Let's either control F8 convolutions with the same `MIOPEN_BETA_API` or do the following:
- Rename existing `MIOPEN_BETA_API` to `MIOPEN_BETA_API_LAYERNORM`
- Rename `MIOPEN_BETA_API_FP8` to `MIOPEN_BETA_API_CONV_F8`. Why:
  - `CONV` because this only controls convolutions
  - `F8` because this controls both FP8 and BF8 datatypes.

Note: Related review thread (unresolved for now) can be found at https://github.com/ROCmSoftwarePlatform/MIOpen/pull/2320#discussion_r1316419544

/cc @JehandadKhan

Reply:
@atamazov I had meant to use simply MIOPEN_BETA_API and default the CMakeLists.txt to defining this variable thusly unconditionally enabling for all sources, and disabling in miopen.h by default. However the implementation of layernorm appears to rely on MIOPEN_BETA_API remaining undefined, because there are compilation errors in driver/layernorm_driver.hpp pending an undefined RAN_GEN function.

Reply:
fixed this

Reply:
[resolved]